paper-dialog-behavior.html

```diff
@@ -205,32 +205,36 @@
     },
 
     _onFocus: function(event) {
-      if (this.modal) {
```

Comment:
Well, the reason I ask is that we have a change handler for when `modal` changes. This suggests that the current implementation handles changes of the `modal` property. We should probably not deviate from the current implementation. WDYT?